server/libs/database/src/entities/user.entity.ts

```diff
@@ -3,32 +3,32 @@ import { Column, CreateDateColumn, Entity, PrimaryGeneratedColumn } from 'typeor
 @Entity('users')
 export class UserEntity {
   @PrimaryGeneratedColumn('uuid')
-  id: string;
+  id!: string;
```

Comment:
Yes exactly, it's called definite assignment assertion. In general we should only be using these in Entities where properties are not null, because TypeORM cannot use a constructor to initialise the values. So we know that after the Entity is "hydrated" by TypeORM, those properties would have its value.